src/app/modules/time-entries/pages/time-entries.component.spec.ts

```diff
@@ -363,12 +364,13 @@ describe('TimeEntriesComponent', () => {
 
       }, shouldRestartEntry: true
     };
+
     component.entryId = '123';
     spyOn(store, 'dispatch');
 
     component.doSave(entryToSave);
 
-    expect(store.dispatch).toHaveBeenCalledWith(new entryActions.RestartEntry(entryToSave.entry));
+    expect(store.dispatch).toHaveBeenCalledWith(new entryActions.UpdateEntry(entryToSave.entry));
```

Comment:
The same as the other comment, just wan to understand why we change from "Restart..." to "Update..."

Reply:
With the new changes it is not necessary to restart the entry when I edit the entries.
Only with the update does not get the error of two inputs in progress

Reply:
Mmm if you say that you don't need to `restart`, why are you setting `shouldRestartEntry` as `true` in this test?
Please consider a review on the whole test.